include/zenoh-pico/net/private/system.h

```diff
@@ -15,15 +15,19 @@
 #ifndef _ZENOH_NET_PICO_SYSTEM_H
 #define _ZENOH_NET_PICO_SYSTEM_H
 
-#if (ZENOH_LINUX == 1) || (ZENOH_MACOS == 1)
+#if (ZENOH_LINUX == 1) || (ZENOH_MACOS == 1) || (ESP32 == 1) || (ARDUINO_ARCH_ESP32 == 1)
 #include "zenoh-pico/net/private/system/unix.h"
 #elif (ZENOH_CONTIKI == 1)
 #include "zenoh-pico/net/private/contiki/types.h"
 #endif
 
 /*------------------ Network ------------------*/
 // @TODO: remove the platform-specific include and data types
+#if (ESP32 == 1) || (ARDUINO_ARCH_ESP32 == 1)
```

Comment:
It would be preferable to use the same naming convention:
ESP32 -> ZENOH_ESP32
ARDUINO_ARCH_ESP32 -> ZENOH_ARDUINO_ESP32